### PR TITLE
RASPA2 direct running problem solution

### PR DESCRIPTION
**Pull request by hajgato**
libraspa2 stub loader confuses the direct command `simulate`, therefore it is deleted after installation.

### CODE REVIEW

**Comment by boegel** [comment]
@hajgato Can you please clarify this a bit more? What problem/errors are you running into exactly without this?

Also, please include a line or two of comments in the easyconfig itself to explain why this is needed, it looks kind of dirty...

**Comment by hajgato** [comment]
@boegel 
The error message without deleting the stub loaders:
```
Process Process-1:
Traceback (most recent call last):
  File "/apps/brussel/ivybridge/software/Python/2.7.12-intel-2016b/lib/python2.7/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/apps/brussel/ivybridge/software/Python/2.7.12-intel-2016b/lib/python2.7/multiprocessing/process.py", line 114, in run
    self._target(*self._args, **self._kwargs)
  File "/apps/brussel/ivybridge/software/RASPA2/2.0.3-intel-2016b-Python-2.7.12/lib/python2.7/site-packages/RASPA2-2.0.3-py2.7-linux-x86_64.egg/RASPA2/raspa2.py", line 119, in _script_subprocess
    libraspa = cdll.LoadLibrary(os.path.join(libraspa_dir, libraspa_file))
  File "/apps/brussel/ivybridge/software/Python/2.7.12-intel-2016b/lib/python2.7/ctypes/__init__.py", line 440, in LoadLibrary
    return self._dlltype(name)
  File "/apps/brussel/ivybridge/software/Python/2.7.12-intel-2016b/lib/python2.7/ctypes/__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: /vsc/apps/brussel/ivybridge/software/RASPA2/2.0.3-intel-2016b-Python-2.7.12/lib/python2.7/site-packages/RASPA2-2.0.3-py2.7-linux-x86_64.egg/RASPA2/simulations/lib/libraspa2.pyc: invalid ELF header
```
The responsible lines are in `raspa2.py`:
```
19 libraspa_file = next(f for f in os.listdir(libraspa_dir) if "libraspa" in f)
...
119     libraspa = cdll.LoadLibrary(os.path.join(libraspa_dir, libraspa_file))
```
I guess, this is the way how the authors handle the possible different extensions for dynamic libraries...

When I download the python wheel, there is no stub loader for `libraspa2.so`
See: https://pypi.python.org/packages/8a/8c/5efbf97988c86f120646e3c575b53b5ce63d09eb6692c11e1a89f288f3a9/RASPA2-2.0.3-cp34-cp34m-macosx_10_5_x86_64.whl#md5=1ff1610a19ac2e27019272e4b5283e6f

Based on those facts, I decided not to make patch for `RASPA2`, but delete the stub loader. 
I have no idea how can I comment this in 1-2lines in the easyconfig.

File: easybuild/easyconfigs/r/RASPA2/RASPA2-2.0.3-intel-2016b-Python-2.7.12.eb [hunk 1]
```diff
@@ -18,6 +18,11 @@ dependencies = [
     ('Python', '2.7.12'),
 ]
 
+to_delete = '%(installdir)s/lib/python%(pyshortver)s/site-packages'
+to_delete += '/RASPA2-%(version)s-py%(pyshortver)s-linux-x86_64.egg'
+to_delete += '/RASPA2/simulations/lib/libraspa2.py*'
+postinstallcmds = ['rm %s' % to_delete]
```
Comment (hajgato) [inline easybuild/easyconfigs/r/RASPA2/RASPA2-2.0.3-intel-2016b-Python-2.7.12.eb]: @boegel Its good! :)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2015.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/2b5fe06fca6bd91836ac96173452cc25 for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2464.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/7f725ccd6ba06fd8cbcbeb4dd77a450a for a full test report.

**Comment by boegel** [comment]
Going in, thanks @hajgato!

**Pull request merged**